### DISCUSSION TITLE
Manfully set max charging power

### DISCUSSION THREAD

**Discussion by devdems** [serkri/SmartEVSE-3#242] [General]
Hi, i know it is written that the station dynamically sets the power based on the connected cable. My question is if we can limit max charging current manually?

**Comment by dingo35** [comment]
On the webserver screen you can set "Override Current"; you can also set this through the REST API, MQTT or the homeassistant integration.

For details see [https://github.com/dingo35/SmartEVSE-3.5/tree/master/docs](url)

**Comment by devdems** [comment]
Thank you.
